src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAYS_PRICE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * ONE_DAYS_PRICE;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
```

Comment:
Instead of using 'else if' after a return statement, simply use an 'if' statement. The 'else' is unnecessary because if the condition in the first 'if' statement is true, the function will return and not proceed to the next 'if' statement.